src/components/NFTEditionSelect.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div :class="rootClasses">
     <table
-      v-if="!isSingleItem && !isAllSoldOut"
+      v-if="!isSingleItem || (isSingleItem && items[0].name) && !isAllSoldOut"
```

Comment:
Confusing, does this means
`(!isSingleItem || (isSingleItem && items[0].name)) && !isAllSoldOut`
or
`!isSingleItem || ((isSingleItem && items[0].name) && !isAllSoldOut)` ?

Reply:
`(!isSingleItem || (isSingleItem && items[0].name)) && !isAllSoldOut`